alacritty/src/cli.rs

```diff
@@ -245,7 +245,7 @@ impl Options {
             // Arg::min_values(1) is set.
             let command = String::from(args.next().unwrap());
             let args = args.map(String::from).collect();
-            options.command = Some(Shell::new_with_args(command, args));
+            options.command = Some(Shell::new_with_args(&command, args));
```

Comment:
This is kind of bad style. Creating something as String, just to pass it to a function as reference which then converts it to a String again.

If this is the only consumer, a `String` parameter would work just fine. Otherwise `Into<String>` is also a better alternative.

Reply:
I was hoping to avoid the OS specific files in this pull, but I guess I can't.